code/game/objects/items/tools/synth_bracer/abilities/integrated/ocular_designator.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/clothing/gloves/synth
```

Comment:
Why bother with deployable binoculars, is this on purpose? Zooming is a property from /obj/item so you could directly bake it into glove without having to drag that around